cmake/DeployQt5.cmake

```diff
@@ -0,0 +1,373 @@
+    set(plugin_release_var "${plugin_var}_RELEASE")
+    set(plugin_debug_var "${plugin_var}_DEBUG")
+    set(plugin_release "${${plugin_release_var}}")
+    set(plugin_debug "${${plugin_debug_var}}")
+    if(DEFINED "${plugin_release_var}" AND DEFINED "${plugin_debug_var}" AND NOT EXISTS "${plugin_release}" AND NOT EXISTS "${plugin_debug}")
+      message(WARNING "Qt plugin \"${plugin}\" not recognized or found.")
+    endif()
+    if(NOT EXISTS "${${plugin_debug_var}}")
+      set(plugin_debug "${plugin_release}")
+    endif()
+
+    if(CMAKE_CONFIGURATION_TYPES OR CMAKE_BUILD_TYPE)
+      install_qt5_plugin_path("${plugin_release}" "${executable}" "${copy}" "${installed_plugin_path_var}_release" "${plugins_dir}" "${component}" "Release|RelWithDebInfo|MinSizeRel")
+      install_qt5_plugin_path("${plugin_debug}" "${executable}" "${copy}" "${installed_plugin_path_var}_debug" "${plugins_dir}" "${component}" "Debug")
+
+      if(CMAKE_BUILD_TYPE MATCHES "^Debug$")
+        set(${installed_plugin_path_var} ${${installed_plugin_path_var}_debug})
+      else()
+        set(${installed_plugin_path_var} ${${installed_plugin_path_var}_release})
+      endif()
+    else()
+      install_qt5_plugin_path("${plugin_release}" "${executable}" "${copy}" "${installed_plugin_path_var}" "${plugins_dir}" "${component}")
+    endif()
+  endif()
+  set(${installed_plugin_path_var} ${${installed_plugin_path_var}} PARENT_SCOPE)
+endfunction()
+
+function(install_qt5_executable executable)
+  set(qtplugins ${ARGV1})
+  set(libs ${ARGV2})
+  set(dirs ${ARGV3})
+  set(plugins_dir ${ARGV4})
+  set(request_qt_conf ${ARGV5})
+  set(component ${ARGV6})
+  if(QT_LIBRARY_DIR)
+    list(APPEND dirs "${QT_LIBRARY_DIR}")
+  endif()
+  if(QT_BINARY_DIR)
+    list(APPEND dirs "${QT_BINARY_DIR}")
+  endif()
+  if(component)
+    set(component COMPONENT ${component})
+  else()
+    unset(component)
+  endif()
+
+  get_filename_component(executable_absolute "${executable}" ABSOLUTE)
+  if(EXISTS "${QT_QTCORE_LIBRARY_RELEASE}")
+    gp_file_type("${executable_absolute}" "${QT_QTCORE_LIBRARY_RELEASE}" qtcore_type)
+  elseif(EXISTS "${QT_QTCORE_LIBRARY_DEBUG}")
+    gp_file_type("${executable_absolute}" "${QT_QTCORE_LIBRARY_DEBUG}" qtcore_type)
+  endif()
+  if(qtcore_type STREQUAL "system")
+    set(qt_plugins_dir "")
+  endif()
+
+  if(QT_IS_STATIC)
+    message(WARNING "Qt built statically: not installing plugins.")
+  else()
+    if(APPLE)
+      get_property(loc TARGET Qt5::QCocoaIntegrationPlugin
+        PROPERTY LOCATION_RELEASE)
+      install_qt5_plugin("${loc}" "${executable}" 0 installed_plugin_paths
+        "${plugins_dir}/platforms" "${component}")
+      list(APPEND libs ${installed_plugin_paths})
+    elseif(WIN32)
+      get_property(loc TARGET Qt5::QWindowsIntegrationPlugin
+        PROPERTY LOCATION_RELEASE)
+      install_qt5_plugin("${loc}" "${executable}" 0 installed_plugin_paths
+        "platforms" "${component}")
```

Comment:
Thanks for the suggestion.  Does packaging work for you if this is changed?

(As you can tell, I lost steam on fixing this packaging issue, which is part of the reason displaz hasn't had a proper release in far too long.)